docs/api/StreamClose.md

```diff
@@ -23,12 +23,15 @@ A Stream handle from a previous call to [StreamOpen](StreamOpen.md).
 
 # Remarks
 
-The application should only close a stream after it has been completely shut down or it was never successfully started. Closing a stream before it has been completely shut down produces **undefined behavior** because clean up of the stream **must** be reflected on the wire with an application specific error code. When the app closes a stream without first shutting down, MsQuic has to guess which error code to use (currently uses `0`) when sending the state change to the peer.
+The application should only close a stream after it has been completely shut down or if it was never successfully started. Closing a stream before it has been completely shut down produces **undefined behavior** because clean up of the stream **must** be reflected on the wire with an application specific error code. When the app closes a stream without first shutting down, MsQuic has to guess which error code to use (currently uses `0`) when sending the state change to the peer.
```

Comment:
I believe that code comment in stream.c is in a block that will abort the entire connection if a stream is closed without being shutdown first. 
This remark here (and possibly also in Streams.md) should mention that this undefined behavior will **terminate the connection**, not just the stream, which seems implied by "[MsQuic] guess[ing] the error code".

Reply:
:) We are going in a loop!

I started with the documentation stating what the comment says, then Nick mentioned the comment is wrong, so I removed the comment and adapted this paragraph.

Github is not helping, not showing comments on previous iterations.

Reply:
yeah, I wish github would show a tombstone of the previous comment or something. Sorry to run you around in circles.